prow/cmd/branchprotector/protect.go

```diff
@@ -36,10 +37,11 @@ import (
 )
 
 type options struct {
-	config    string
-	jobConfig string
-	confirm   bool
-	github    flagutil.GitHubOptions
+	config              string
+	jobConfig           string
+	confirm             bool
+	verifyAuthorisation bool
```

Comment:
Please follow the convention of the rest of the org in spelling this word (and be consistent within your PR):

https://github.com/kubernetes/test-infra/search?q=org%3Akubernetes+authorization&unscoped_q=org%3Akubernetes+authorization

Reply:
Thanks, I have updated the PR (used `verifyRestrictions` to match the feature flag).